specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/stable/2021-03-07/phonenumbers.json

```
@@ -0,0 +1,951 @@
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Defines the update capabilities request.",
+            "schema": {
+              "$ref": "#/definitions/PhoneNumberCapabilitiesRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The service has accepted the request and will start processing later. It will return 'Accepted' immediately and include an 'Operation-Location' header. Client side should further query the operation status using the URL specified in this header.",
+            "schema": {
+              "$ref": "#/definitions/AcquiredPhoneNumber"
+            },
+            "headers": {
+              "Operation-Location": {
+                "description": "URL to query for status of the operation.",
+                "type": "string"
+              },
+              "Location": {
+                "description": "Url to retrieve the final result after operation completes.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Microsoft.CommunicationServicesCommon/preview/2020-11-19-preview1/common.json#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Update phoneNumber capabilities": {
+            "$ref": "./examples/UpdatePhoneNumberCapabilities.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PhoneNumberCapabilities": {
+      "description": "Capabilities of a phone number.",
+      "required": [
+        "calling",
+        "sms"
+      ],
+      "type": "object",
+      "properties": {
+        "calling": {
+          "description": "Capability value for calling.",
+          "enum": [
+            "none",
+            "inbound",
+            "outbound",
+            "inbound+outbound"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberCapabilityValue",
+            "modelAsString": true
+          }
+        },
+        "sms": {
+          "description": "Capability value for SMS.",
+          "enum": [
+            "none",
+            "inbound",
+            "outbound",
+            "inbound+outbound"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberCapabilityValue",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PhoneNumberSearchRequest": {
+      "description": "Represents a phone number search request to reserve some phone numbers.",
```

Maybe

```suggestion
      "description": "Represents a phone number search request to find phone numbers. Found phone numbers are temporarily reserved for a following purchase.",
```

Or "held" instead of "reserved" to match the description for `searchExpiresBy` below.